addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -72,6 +73,54 @@ export class AnalyticDistribution extends Component {
         this.allPlans = [];
         this.lastAccount = this.props.account_field && this.props.record.data[this.props.account_field] || false;
         this.lastProduct = this.props.product_field && this.props.record.data[this.props.product_field] || false;
+
+        this.activeTag = null;
+        this.selectCreateIsOpen = false;
+        const selectCreate = useSelectCreate({
+            resModel: 'account.analytic.account',
+            activeActions: useActiveActions({
+                fieldType: "many2many",
+                crudOptions: {
+                    link: true,
+                    create: false,
+                },
+                getEvalParams: () => ({
+                    readonly: false,
+                }),
+            }),
+            onSelected: async (resIds) => {
+                const analytic_accounts = await this.fetchAnalyticAccounts([["id", "in", resIds]]);
+                // modify the selected tag
+                this.activeTag.analytic_account_id = analytic_accounts[0].id;
+                this.activeTag.analytic_account_name = analytic_accounts[0].name;
+                this.setFocusSelector(`.tag_${this.activeTag.id} .o_analytic_percentage`);
+                if (analytic_accounts.length > 1) {
+                    const plan_id = analytic_accounts[0].root_plan_id[0];
+                    // remove the autofill line
+                    this.list[plan_id].distribution = this.list[plan_id].distribution.filter((t) => !!t.analytic_account_id);
+                    for (let i = 1; i < analytic_accounts.length; i++) {
```

Comment:
```suggestion
                    for (let account of analytic_accounts.slice(1)) {
```

It would even be better if we could remove the `slice` and the `if (analytic_accounts.length > 1)` above